shopfloor/services/forms/picking_form.py

```diff
@@ -46,7 +46,10 @@ class ShopfloorPickingFormValidator(Component):
     _usage = "form_edit_stock_picking.validator"
 
     def get(self):
-        return {}
+        return {
```

Comment:
I'd move this to the mixin.

Reply:
When I moved this to the mixin I get a NotImplementedError and I don't know why ?

Reply:
It is :green_apple: now